app/controllers/comments_contoller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+    @user = @post.author
+  end
+
+  def create
+    @comment = Comment.new(comments_params)
+    @comment.author = current_user
+    @comment.post = Post.find(params[:post_id])
+    if @comment.save
+      redirect_to user_post_path(@comment.post.author, @comment.post)
+    else
+      render :new
+    end
```

Comment:
- [ ] When I try to create a comment from the form and click on create nothing happens so please make sure to fix that issue so that the comment gets created without prob